src/main/java/command/Event.java

```diff
@@ -0,0 +1,26 @@
+package main.java.command;
+
+import main.java.classes.Task;
+
+import java.util.Date;
+
+/**
+ * Event inherits from Task. It encapsulates an event with an additional String at.
+ * Event is specified by [E].
+ */
+public class Event extends Task {
+    protected Date at;
+    public Event (String description, Date at) {
```

Comment:
There is a missing line break between the variable declaration and constructor. You might have missed it out by accident.